apiserver/openapi.json

```diff
@@ -13,7 +13,7 @@
     "url" : "http://template/v1"
```

Comment:
This dummy url should also be name like the app: "http://thingdust/v1"

Reply:
The url has been updated.